tests/test_client_wait.py

```diff
@@ -0,0 +1,22 @@
+from unittest.mock import MagicMock
+
+import boto3
+import pytest
+from moto import mock_batch, mock_iam
+from prefect import flow, task
+
+from prefect_aws.client_wait import _load_prefect_waiter
+
+
+def mock_create_waiter_with_client(*args, **kwargs):
+    return "waiter"
+
+
+def test_load_prefect_waiter(monkeypatch):
+    monkeypatch.setattr(
+        "prefect_aws.client_wait.create_waiter_with_client",
+        mock_create_waiter_with_client,
+    )
+    mock_boto_client = MagicMock()
+    result = _load_prefect_waiter(mock_boto_client, "batch", "JobExists")
+    assert result == "waiter"
```

Comment:
We want to add a test for `client_waiter` too. Ensure that the if / else clauses are covered in your test, i.e. try to write a test that enters those clauses. Example: since one of the clauses is `if waiter_definition: ...`, to enter the clause, simply provide the waiter definition.